cmd/init.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2017 The Kedge Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"text/template"
+
+	"github.com/spf13/cobra"
+)
+
+type templateData struct {
+	Name  string
+	Image string
+	Ports []string
+}
+
+var (
+	fileName, image, name string
+	port                  []string
+)
+
+const (
+	boilerplate = `name: {{.Name}}{{"\n"}}containers:{{"\n"}}- image: {{.Image}}{{"\n"}}{{if .Ports}}services:{{"\n"}}- ports:{{block "list" .Ports}}{{"\n"}}{{range .}}{{println "  - port:" .}}{{end}}{{end}}{{end}}`
```

Comment:
can this be made multiline? It's hard to read and follow all of it in single file!